workinggroups/README.md

```diff
@@ -2,19 +2,27 @@
 
 ## Introduction
 
-The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
+CNCF Working Groups (WGs) serve several purposes, for example to study and report on a particular question, to make recommendations based on their findings, or to complete a specific work package. The intention of WGs is _not_ to host a full project or specification. The work result of a WG may include a new project proposal, landscape, whitepaper report detailing their findings, or similar.
 
-## Process
+WGs must be created under the umbrella of a [CNCF Technical Advisory Group](https://github.com/cncf/toc/tree/main/tags) (TAG), and sponsored by a CNCF TOC member. In case of disagreement within TOC, a supermajority vote may be called 
+
+WGs can be shut down when they determine that they have reached their stated goals, by consensus within their umbrella TAG, or a supermajority vote within the TOC.
```

Comment:
```suggestion
WGs can be shut down when the working group and TAG, with concurrence of the TOC sponsor, determines that they have reached their stated goals, by consensus within their umbrella TAG, or by a supermajority vote within the TOC.
```

Reply:
clarifying here that the Working Group and the TAG need to agree that they've met their stated goals (forcing the WG to have the discussion to the TAG for closure), and then concurrence with the TOC sponsor for closure (this also ensures the TOC sponsor is aware of what that deliverable from the goals is), _as an additional option_ to the TAG consensus and TOC supermajority vote. There may be occasions where the working group fizzles out and the TAG itself (through lack of participation or progress) chooses to dissolve/disband/shutdown the group.

Reply:
@raravena80 Each TAG has a sponsor as part of its support system. This PR is to clarify that WG should be formed within an established TAG to ensure we are following established processes around governance, process, etc.. Does this clarify your questions?

Reply:
@raravena80 are you talking about creating or disbanding a WG in your comment?